gcc/common.opt

```diff
@@ -2230,6 +2230,10 @@ fomit-frame-pointer
 Common Var(flag_omit_frame_pointer) Optimization
 When possible do not generate stack frames.
 
+fforce-omit-frame-pointer
```

Comment:
I have to admit this feels a little gross, but I guess this is because we don't want to manually patch all software in the world for to add `-fno-omit-frame-pointer` ala what Fedora and co. have done. Just wondering if we should try to make this similar versus yet another custom op.

It's fine either way.